src/role_atkeeper.js

```diff
@@ -9,17 +9,14 @@
 
 roles.atkeeper = {};
 
-roles.atkeeper.getPartConfig = function(room, energy, heal) {
-  var parts = [MOVE, RANGED_ATTACK, MOVE, HEAL];
-  return room.getPartConfig(energy, parts);
-};
-
-roles.atkeeper.energyRequired = function(room) {
-  return Math.min(room.getEnergyCapacityAvailable(), 3430);
-};
-
-roles.atkeeper.energyBuild = function(room) {
-  return Math.min(room.getEnergyCapacityAvailable(), 3430);
+roles.atkeeper.settings = {
+  parts: {
+    layout: 'MMRH'
+  },
+  energy: {
+    minEnergyStored: 3430,
+    maxEnergyUsed: 3430
+  }
```

Comment:
@Nevrdid asked a couple of times for that :-)

IMHO it is a good to have the role config at one place. If we removed a role, just delete this file. If you create a new role, create a file and fill it. If you search for something role specific, you will find it in that file.

If there are defaults (I don't think there are) we could use some default config and inherit from there.